### PR TITLE
[RAPPS] Romanian translation update

### PR DESCRIPTION
**Pull request by Fraizeraust**
## Purpose & Changes

Here is the patch with the latest improvements for the Romanian language. Header file is also added for better information about the translation file itself.

### CODE REVIEW

File: base/applications/rapps/lang/ro-RO.rc [hunk 1]
```diff
@@ -1,4 +1,13 @@
-/* Ștefan Fulea (stefan dot fulea at mail dot md) */
+/*
+ * PROJECT: ReactOS Applications Manager (RAPPS)
+ * FILE:    base/applications/rapps/lang/ro-RO.rc
+ * PURPOSE: Romanian Translation for RAPPS
+ * TRANSLATORS:
+ *          Ștefan Fulea (stefan dot fulea at mail dot md)
+ *          Bișoc George (fraizeraust99@gmail.com)
+ * HISTORY:
+ *          29/12/2017 -- Update by Fraizeraust alias Bișoc George (fraizeraust99@gmail.com)
```
Comment (ThFabba) [inline base/applications/rapps/lang/ro-RO.rc]: This is not necessary, that's what "git log" is for :)

Comment (Fraizeraust) [reply base/applications/rapps/lang/ro-RO.rc]: Must I delete the whole header or just the "HISTORY" parth, Thomas Faber?

Comment (ThFabba) [reply base/applications/rapps/lang/ro-RO.rc]: The rest of the header seems fine.

**Comment by Fraizeraust** [comment]
A hot-fix has been committed. Thank you for the reply, Thomas Faber!

**Comment by sanchaez** [comment]
@Fraizeraust please specify what name and email you want to use for the commits.

**Comment by Fraizeraust** [comment]
My real name's Bișoc George (it is already there on my profile details anyways) and E-mail is fraizeraust99@gmail.com

**Comment by sanchaez** [comment]
Thanks. Just check your git settings :)

**Comment by sanchaez** [comment]
Closed?? Anyway, it's merged in https://github.com/reactos/reactos/commit/409d166c2c979464ba041184f8fc7e37127ab8bc. Thanks!

**Pull request closed without merging**